specification/billing/resource-manager/Microsoft.Billing/preview/2020-11-01-preview/billingPromotions.json

```diff
@@ -172,6 +172,42 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/promotions": {
+      "get": {
```

Comment:
>get [](http://example.com/codeflow?start=7&length=3)

The "PromotionsCheckEligibilityResponse" doesn't represent a list of ARM Resources. Collection GET should only be used to return valid ARM Resources. What is your scenario here?

Reply:
Please also refer to the operation id "Promotion_CheckEligibility" in the this swagger, both these two operations are basically doing the same thing(check eligibility of promotion) except that "Promotions_CheckEligibility" validates the subscriptions to see if eligible for any promotion. So it would make more sense both of these operations are in "GET"